src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
```

Comment:
What does this mean? I take the height header from the layout and it is 60px.

Reply:
<img width="799" alt="image" src="https://user-images.githubusercontent.com/70220667/233565362-d2ff473c-4440-4de1-baae-80ab0937030b.png">


 "Do not hard code height, use padding instead" suggests that instead of setting a fixed height of 60px for the header element, it is better to use padding to create the desired spacing. This is because using padding allows the height of the header to adjust automatically based on its content, which can be more flexible and responsive to different screen sizes and device types.

Reply:
Ok